src/settings.pyj

```diff
@@ -85,6 +85,11 @@ options_list = {
                            "type": "bool",
                            "default": True},
 
+    "initiative-with-dex-score": {"title": "Add dexterity score tiebreaker to initiative roll.",
```

Comment:
I would rename the setting into `initiative-tiebreaker` rather than `initiative-with-dex-score` (what it does, rather than how it does it) and adjust the title accordingly. I'm not super good with naming stuff, but my suggestion would be : 
- title: Add tie breaker to initiative rolls.
- description: Adds the dexterity score as a decimal to initiative rolls to break any initiative ties.